src/components/Map/Map.tsx

```diff
@@ -188,65 +188,96 @@ class Map extends React.Component<MapProps> {
       }
     };
 
+    /**
+     * This function is called after all the markers have been placed on the map.
+     * It uses the `updateSelectedSensor` function from the props to set a default
+     * starting sensor for the home page to display.
+     */
+    const setDefaultSensor = () => {
+      // By default, select a sensor to display:
+      if (map) {
+        const markers = map.getObjects();
+        // If there is an object in the list
+        if (markers.length > 0) {
+          const firstSensor = markers[0];
+          if (firstSensor instanceof H.map.Marker) {
+            const data = firstSensor.getData();
+            const selectedSensor: SelectedSensor = {
+              purpleAirId: data.purpleAirId,
+              sensorDocId: data.sensorDocId,
+              name: data.name,
+              aqi: data.aqi,
+              isValid: data.isValid,
+              lastValidAqiTime: data.lastValidAqiTime,
+            };
+            this.props.updateSelectedSensor(selectedSensor);
```

Comment:
For some reason when the page first loads, the selected sensor doesn't have the bold black border on it